tools/ci-ts/test-helpers/chainlinkClient.test.ts

```diff
@@ -0,0 +1,129 @@
+import * as m from './__mocks__/docker'
```

Comment:
We shouldn't need to import the mock explicitly, jest should do it for us: https://jestjs.io/docs/en/manual-mocks#examples